### PR TITLE
create redshift resource in dev

### PR DESCRIPTION
**Pull request by calvinnhieu**

### CODE REVIEW

File: terraform/modules/matrix-service/infra/redshift.tf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+resource "aws_redshift_cluster" "default" {
+  cluster_identifier = "dcp-matrix-service-cluster-${var.deployment_stage}"
+  database_name      = "${var.deployment_stage}"
```
Comment (parthshahva) [inline terraform/modules/matrix-service/infra/redshift.tf]: please namespace the database_name with matrix-service as well

File: terraform/envs/dev/terraform.tfvars.example [hunk 2]
```diff
@@ -3,3 +3,5 @@ account_id = ""
 aws_region = ""
 converter_cluster_ami_id = ""
 vpc_cidr_block = ""
+redshift_username = ""
```
Comment (parthshahva) [inline terraform/envs/dev/terraform.tfvars.example]: Where are the tfvars stored now? Is this change made in this PR?

Comment (calvinnhieu) [reply terraform/envs/dev/terraform.tfvars.example]: In the S3 terraform bucket, together with the tfstate files

Changes:
https://github.com/HumanCellAtlas/matrix-service/commit/0fc0cb4890f99874f86b55916a8dbc8719ccd874

File: terraform/modules/matrix-service/infra/redshift.tf (hunk 1 above)
Comment (calvinnhieu) [reply terraform/modules/matrix-service/infra/redshift.tf]: Will be named matrix_service_*, does not allow '-'

Comment (parthshahva) [reply terraform/modules/matrix-service/infra/redshift.tf]: sounds good

File: terraform/envs/dev/terraform.tfvars.example (hunk 2 above)
Comment (parthshahva) [reply terraform/envs/dev/terraform.tfvars.example]: got it. This pattern looks familiar ;)

**Pull request merged**